.github/workflows/cron.yml

```diff
@@ -17,14 +17,17 @@ jobs:
 
       - name: Update
         run: |
+          rm -rf *.json
```

Comment:
Is this line necessary? We are overwriting these files anyways.

Reply:
I put that there because I figured we should remove the existing `releases.json` file. We could just manually push a commit to remove that, though, so we could take it out.